qiita_pet/support_files/doc/source/admin/portals.rst

```diff
@@ -0,0 +1,72 @@
+.. _portals:
+
+.. index:: portals
+
+User Portals
+============
+
+User portals allow a user to cordon off specific studies and analyses into their
```

Comment:
This line shows in bold while the rest doesn't:

<img width="741" alt="screen shot 2015-07-05 at 4 30 19 pm" src="https://cloud.githubusercontent.com/assets/2501478/8513907/302f8aba-2333-11e5-932b-7a26f8129f74.png">

I *think* is because the other lines have a space on front.